src/index.ts

```diff
@@ -0,0 +1,9 @@
+import { Manager, MCEvent } from '@managed-components/types'
+
+export const handler = (event: MCEvent) => {
+  event.client.execute(`zaraz.i(\`${escape(event.payload.htmlCode)}\`);`)
```

Comment:
We should relay on a special `zaraz.i` call here, we need to make it work without it (if that means replicating zaraz.i, that's fine